python-sdk/nuscenes/eval/tracking/data_classes.py

```diff
@@ -22,8 +25,12 @@ def __init__(self,
                  metric_worst: Dict[str, float],
                  num_thresholds: int):
 
-        assert set(class_range.keys()) == set(TRACKING_NAMES), "Class count mismatch."
-
+        assert set(class_range.keys()) == set(tracking_names), "Class count mismatch."
+        global TRACKING_NAMES
```

Comment:
why you need to define it as `global` rather than  just `self.tracking_names`

Reply:
https://github.com/nutonomy/nuscenes-devkit/blob/42219be5fb953f82c5bcff80b26663e5ed416291/python-sdk/nuscenes/eval/tracking/data_classes.py#L283
It is needed here, and I donot want to change `TrackingBox` to include extra things.

Reply:
okay, then it's define this global variable at the beginning of this file, since it's shared by the 2 classes.